EIPS/draft_hardware_and_bandwidth_recs.md

```diff
@@ -0,0 +1,148 @@
+
+## Motivation
+
+Clear system specifications are crucial for:
+
+- Ensuring meaningful benchmark comparisons across different implementations
+- Enabling informed decision-making about protocol upgrades and their hardware implications
+- Providing clear guidance for node operators with respects to the future
+
+Without a shared understanding of target hardware specifications:
+
+- Benchmark results lose significance due to inconsistent testing environments
+- Decision-making becomes challenging for implementation choices, as performance characteristics are heavily hardware dependent
+- Network participants lack clear guidance for hardware investments
+
+## Specification
+
+### Recommended hardware and bandwidth specifications
+
+| Node type | Storage | RAM | CPU Cores | CPU Single Thread/Multithread rating | Download/Upload speed |
+| -------- | -------- | -------- | --------| -------- |--------|
+| Full node     | 4TB     | 32GB    | 4 cores/8 threads  | 1000 / 3000 | 50 Mbps / 15 Mbps |
+| Attester      | 4TB     | 64GB    | 8 cores/16 threads | 3500 / 25000 | 50 Mbps / 25 Mbps |
+| Block builder | 4TB     | 64GB    | 8 cores/16 threads | 3500 / 25000 |  100 Mbps / 50 Mbps |
+
+### Recommended Prebuilds (Attester and Block builder)
+
+#### Minisforum UM790 Pro
+
+We recommend the Minisforum UM790 Pro with the following modifications:
+
+- Minisforum UM790 Pro
+- 8 cores
+- 16 threads
+- The barebones model will need to be updated to include 32GB RAM and 4TB storage
+- CPU Rating
+  - Single Thread: 3903
+  - Multithread: 30367
+
+#### ASUS NUC 14 Pro
+
+- 16 cores (6 performance + 8 efficiency cores + 2 Low power efficiency cores)
+- 22 threads
+- CPU Rating
+  - Single Thread: 3520
+  - Multithread: 25158
+
+### Recommended Prebuild (Full node)
+
+#### Raspberry Pi 5
+
+- 4 core
+- 4 threads
+- CPU Rating
+  - Single thread: 1487
+  - Multithred: 3428
+
+## Rationale
+
+### Storage
+
+For storage, we recommend the following SSDs:
+
+- Kingston KC3000
+- Kingston Fury Renegade
+- Samsung 990 Pro
+- Seagate Firecuda 530
+- Teamgroup MP44
+- WD Black SN850X
+
+In particular, we recommend purchasing NVMe M.2 instead of SATA as NVMe has higher throughput.
+
+As of January 2025, the storage requirements for all node types are dominated by the need to store block history.
+A 2TB disk can be used, however this is not recommended due to the rate of history growth and the uncertainty as to when EIP4444 will be complete.
+
+### RAM
+
+RAM/memory is dominated by state cache. As of January 2025, it is possible to run a full node with 16GB of RAM, however this has been known to not work with all combinations of EL and CL clients in the past.
+
+On 32GB vs 64GB; 32GB works right now, however we recommend 64GB as [preliminary benchmarks](https://hackmd.io/@han/bench-hash-in-snark) have shown that zk-STARKS can consume a significant amount of memory and the difference in cost relative to the entire hardware setup for a validator is insignificant.
```

Comment:
I saw that EIPs 2926, 2938, 3298, 3416 and 3607 also used hackmd links however the contents of them can easily be changed, so I wonder if there is a rule about using them?